docs/source/querying.rst

```diff
@@ -0,0 +1,142 @@
+For example, we can ask for a list of all `predictors` associated with the the task ``500daysofsummer`` directly:
+
+.. doctest::
+
+   >>> predictors = neuroscout.predictors.get(dataset_name='NaturalisticNeuroimagingDatabase', task_name='500daysofsummer')
+   >>> [p['name'] for p in predictors][0:5] # Print first 5 predictor names
+   ['sharpness', 'tool', 'subtlexusfrequency_FREQcount', 'subtlexusfrequency_CDcount', 'subtlexusfrequency_FREQlow']
+
+
+Under the hood, `pyNS` looks up the ``dataset_id`` and ``task_id`` for the given ``dataset_name`` and ``task_name`` and then uses these to lookup the ``run_id`` for the given `run`.
+
+-------------------------------------------------
+Getting the data: querying for `predictor_events`
+-------------------------------------------------
+
+An important aspect of `pyNS` is the ability to retrieve moment by moment events for specific predictors.
+
+The simplest way is to simply use ``predictor_id`` to query for a specific Predictor, for a specific ``run_id``:
+
+.. doctest::
+
+   # First two events for Predictor
+   >>> neuroscout.predictor_events.get(predictor_id=40254, run_id=1581)[0:2]  
+   [{'duration': 1.0, 'onset': 0.0, 'predictor_id': 40254, 'run_id': 1581, 'value': '0.03137254901960784'}, {'duration': 1.0, 'onset': 1.0, 'predictor_id': 40254, 'run_id': 1581, 'value': '0.0196078431372549'}]
+
+However, as before, we can make this simpler by taking advantage of pyNS's convenience features, and querying using the names directly.
+Let's try looking up a `Predictor` named `speech` for the task `MerlinMovie`:
+
+.. doctest::
+
+   >>> neuroscout.predictor_events.get(predictor_name='speech', dataset_name='SherlockMerlin', task_name='MerlinMovie')[0:2]
+   [{'duration': 0.30100000000000016, 'onset': 72.422, 'predictor_id': 12725, 'run_id': 134, 'value': '1'}, {'duration': 0.30100000000000016, 'onset': 72.422, 'predictor_id': 12725, 'run_id': 117, 'value': '1'}]
+
+.. note::
+   `PredictorEvents` are primarily associated with ``run_id`` to allow for maximum design flexibility, such as each subject seeing a different stimulus.
+   As such, the above results will contain all event timepoints for all subjects/runs for that Predictor.
+   However, in many cases all subjects will have seen the same movie, in which case you can simply use the events for a single subject as reference.
+
+
+------------------------------------------
+Friendly outputs to pandas DataFrames
+------------------------------------------
+
+You can easily convert any query result to a pandas DataFrame. Simply pass the argument ``output_type='df'`` to the query.
+This is particularly useful for `PredictorEvents`, as the are naturally represented as a pandas DataFrame`.
+
+::
+
+   >>> neuroscout.predictor_events.get(predictor_name='speech', dataset_name='Sherlock_Merlin', task_name='MerlinMovie')
```

Comment:
`output_type='df'` should be passed here?

Reply:
It's an optional argument